index.js

```diff
@@ -565,8 +582,33 @@ class Squeakquel extends Datastore {
         }
 
         return table.findAll(findParams)
-            .then(items => Promise.all(items.map(item =>
-                decodeFromDialect(this.client.getDialect(), item, model))));
+            .then((items) => {
+                if (containsOp) {
+                    const itemsFiltered = [];
+
+                    items.forEach((item) => {
```

Comment:
what does `items` container ? is it entire data set ? does this mean, entire filtering is now happening on API ?

Reply:
Well this part needs some discussion. I was initially thinking of using a flag to denote whether a pipeline has subscribed scms. But this key looked redundant because it already had 2 keys `scmUrls` and `scmUrlsWithActions` identifying such. This another boolean flag would only be required for querying.

Reply:
And since sqlite or mysql doesn't support filtering from array, this filtering needed to be shifted to the API.